keyboards/blu/vimclutch/vimclutch.c

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2023 Brandon Lu
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "quantm.h"
```

Comment:
Oops, typo on my part.
```suggestion
#include "quantum.h"
```

Reply:
I noticed this during my compilation as well, but forgot to commit it before asking for the re-review.